src/ua/data_types/browse_direction.rs

```diff
@@ -2,22 +2,32 @@ use open62541_sys::UA_BrowseDirection;
 
 crate::data_type!(BrowseDirection);
 
+crate::enum_variants!(
+    BrowseDirection,
+    UA_BrowseDirection,
+    [FORWARD, INVERSE, BOTH, INVALID],
+);
+
 impl BrowseDirection {
+    #[deprecated(note = "use `Self::FORWARD` instead")]
```

Comment:
Mentioning this breaking change in the changelog is sufficient. We don't need to maintain deprecated API elements in a pre-1.0 crate that hasn't been announced anywhere ;)

Reply:
Simply bump the version to v0.5 and drop the deprecated elements.